apps/src/sites/studio/pages/projects/index.js

```diff
@@ -67,26 +71,54 @@ $(document).ready(() => {
       publicGallery);
   });
 
+  // We're going to run an A/B experiment to compare (un)publishing from the
+  // quick actions dropdown and from a button in the published column.
+  // 50% of users will see the chevron variant.
+  // The other 50% of users will see the button variant.
+  // TODO (Erin B.) delete the duplicate table when we
+  // determine the experiment outcome.
   if (experiments.isEnabled(experiments.REACT_PROJECTS_TABLE)) {
-    const personalProjectsUrl = `/api/v1/projects/personal`;
-
-    $.ajax({
-      method: 'GET',
-      url: personalProjectsUrl,
-      dataType: 'json'
-    }).done(personalProjectsList => {
-      store.dispatch(setPersonalProjectsList(personalProjectsList));
-
-      ReactDOM.render(
-        <Provider store={store}>
-          <PersonalProjectsTable
-            canShare={projectsData.canShare}
-            publishMethod="chevron"
-          />
-        </Provider>,
-       document.getElementById('react-my-projects')
-      );
-    });
+
+    if (experiments.isEnabled(experiments.CHEVRON_PUBLISH_EXPERIMENT)) {
```

Comment:
@balderdash this is the line I'm unsure about - how do I check if the user is in the publish-chevron experiment or not?

Reply:
This is how you do it!